src/App.js

```diff
@@ -10,10 +11,29 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+export class App extends Component {
+  state = {
+    selectedTabId: 'tab-1',
+  };
 
-export default App;
+  selectedTab = (tabId) => {
```

Comment:
```suggestion
  selectTab = (tabId) => {
```
в названии функции должен быть глагол - что делаем - выбираем табу